README.md

```diff
@@ -1 +1,343 @@
 placeholder
+
+# Chef Core
+[![Gem Version](https://badge.fury.io/rb/chef-chef-core.svg)](https://badge.fury.io/rb/chef-core)
+
+Chef Core provides low-level tools for building Chef workflows. It contains the subset of
+functionality that we extracted from `chef-run` for use across CLI tools.
+
+This gem still has rough edges. Work is in progress on the following:
+
+- Class/API documentation
+- Reducing complexity of error handling
+- Usage examples
+
+For real-life usage of `chef-core` components, see the [chef-run](https://github.com/chef/chef-apply)
+implementation.
+
+## Connectivity
+
+`chef-core` provides an interface to Train via `ChefCore::TargetHost`.  This is a light wrapper
+around Train that encapsulates the process of setting up and acquiring the underlying connection; and has
+platform-specific awareness of how to perform common operations on the
+remote host such as file delete, package installation, directory and tempdir creation.
+It currently supports Linux and Windows platforms using ssh or winrm protocols. Connection options can
+be found in [https://github.com/inspec/train.git](the train repository).
```

Comment:
This link needs to be fixed. It's not showing up as a proper link at the moment.